zstash/create.py

```diff
@@ -52,15 +53,22 @@ def create():
     if hpss != "none":
         # config.hpss is not "none", so we need to
         # create target HPSS directory
+        machine: str = get_machine()
+        if machine == "chrysalis":
+            hpss_command = "archive"
+        elif machine == "cori":
+            hpss_command = "hsi -q"
+        else:
+            raise ValueError("Invalid machine={}".format(machine))
         logger.debug("Creating target HPSS directory")
-        mkdir_command: str = "hsi -q mkdir -p {}".format(hpss)
+        mkdir_command: str = "{} mkdir -p {}".format(hpss_command, hpss)
```

Comment:
@golaz As far as I can tell, if a file is in `/home/<user>/zstash_archive_test` then `archive` will store it in `/gpfs/fs1/home/<user>/zstash_archive_test/`. I don't see any way to set the HPSS file path to be anything different. This seems like a major obstacle.

Reply:
You want the path to be under some generic user or no user at all?